Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,26 @@
+require "formula"
+require "language/go"
+
+class Vegeta < Formula
+  homepage "https://github.com/tsenart/vegeta"
+  url "https://github.com/tsenart/vegeta/archive/v5.5.2.zip"
+  version "5.5.2"
+  sha1 "f7f61a9512fe09490ea1e0114bb0bbcf450c9a52"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/bmizerany/perks" do
+    url "https://github.com/bmizerany/perks.git",
+      :revision => "6cb9d9d729303ee2628580d9aec5db968da3a607"
+  end
+
+  def install
+    mkdir_p buildpath/"src/github.com/tsenart/"
+    ln_s buildpath, buildpath/"src/github.com/tsenart/vegeta"
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system 'go', 'build', '-o', 'vegeta'
+    bin.install "vegeta"
+  end
```

Comment:
No problem, I prefer to fix this while it's fresh :)

On Mon, Feb 9, 2015 at 11:15 PM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/vegeta.rb
> <https://github.com/Homebrew/homebrew/pull/36685#discussion_r24370722>:
>
> > +  depends_on "go" => :build
> > +
> > +  go_resource "github.com/bmizerany/perks" do
> > +    url "https://github.com/bmizerany/perks.git",
> > +      :revision => "6cb9d9d729303ee2628580d9aec5db968da3a607"
> > +  end
> > +
> > +  def install
> > +    mkdir_p buildpath/"src/github.com/tsenart/"
> > +    ln_s buildpath, buildpath/"src/github.com/tsenart/vegeta"
> > +    ENV["GOPATH"] = buildpath
> > +    Language::Go.stage_deps resources, buildpath/"src"
> > +
> > +    system 'go', 'build', '-o', 'vegeta'
> > +    bin.install "vegeta"
> > +  end
>
> Thanks for the quick fixes, Appreciate that :)
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/36685/files#r24370722>.
>